buildconfig/Jenkins/Conda/nightly_build_and_deploy.jenkinsfile

```diff
@@ -111,7 +111,10 @@ pipeline {
               }
             }
             agent { label "${PLATFORM}" }
-            options { timestamps () }
+            options {
```

Comment:
This looks good and matches what the documentation suggests is the right way to go.

Can I just be picky about the indentation from a readability point of view then can we indent the lines inside the block and leave the closing brace under the `o` as:

```groovy
options {
  timestamps()
  retry(3)
}
```